curriculum/challenges/english/07-scientific-computing-with-python/learn-the-bisection-method-by-finding-the-square-root-of-a-number/65ef190c6b51e9b5a5f7ed29.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 65ef190c6b51e9b5a5f7ed29
+title: Step 2
+challengeType: 20
+dashedName: step-2
+---
+
+# --description--
+
+Give the `square_root_bisection` method the following parameters:
+
+- `square_target`: The number for which you want to find the square root.
+- `tolerance` (optional): The acceptable difference between the square of the approximate root value and the actual target value (default is `1e-7`).
```

Comment:
Regarding the tolerance, if we haven't done it yet, maybe we should mention what the notation `1en` means.